static/specification.html

```diff
@@ -59,384 +58,130 @@
     </section>
     <section id="terminology">
-      <h2>Terminology</h2>
+      <h2>
+        Terminology
+      </h2>
       <dl>
-        <dt><dfn>User</dfn>:</dt>
-        <dd>the party who is accessing the web-monetized website.</dd>
-        <dt><dfn>Website</dfn>:</dt>
-        <dd>the web-monetized website.</dd>
-        <dt><dfn data-lt="provider">Web Monetization provider</dfn>:</dt>
-        <dd>the party making payments on behalf of the user.</dd>
-        <dt><dfn>Web Monetization receiver</dfn>:</dt>
-        <dd>the party receiving payments on behalf of the website.</dd>
-        <dt><dfn>User agent</dfn>:</dt>
-        <dd>the web browser used by the user.</dd>
+        <dt>
+          <dfn data-lt="provider">Web Monetization provider</dfn>:
+        </dt>
+        <dd>
+          The party making payments on behalf of the user. A Web Monetization
+          provider can, for example, make use of the Simple Payment Setup
+          Protocol (<abbr title="Simple Payment Setup Protocol">SPSP</abbr>) on
+          top of the Interledger Protocol (<abbr title=
+          "">ILP</abbr>)/[[[STREAM]]] to provide a high-level way to send money
+          and data.
+        </dd>
+        <dt>
+          <dfn>Web Monetization receiver</dfn>:
+        </dt>
+        <dd>
+          The party receiving payments on behalf of the website.
+        </dd>
+        <dt>
+          <dfn>Payment end-point</dfn>:
+        </dt>
+        <dd>
+          A [=URL=] to an [[[OpenPayments]]] conforming HTTP end-point.
+        </dd>
       </dl>
     </section>
     <section id="goals" class="informative">
-      <h2>Goals</h2>
+      <h2>
+        Goals
+      </h2>
       <ul>
-        <li>Should be simple for webmasters to use in their site.</li>
-        <li>
-          Backend infrastructure should be optional; should be usable on a
-          static site.
+        <li>Suitable for static HTML documents, without requiring backend
+        infrastructure.
+        </li>
+        <li>Doesn't require user interaction, but user retains control over
+        which sites they monetize and how much they spend.
         </li>
-        <li>Interaction with the user should be optional.</li>
-        <li>
-          The user retains control over how much to spend, and which sites to
-          support.
+        <li>Developers have a way to associate payments with their users, in
+        order to provide a range of experiences.
         </li>
-        <li>
-          Should give advanced webmasters a way to associate payments with their
-          users, in order to unlock premium experiences.
+        <li>Pays continuously as the user consumes content.
         </li>
-        <li>Should pay continuously as the user consumes content.</li>
-        <li>
-          Should be compatible with existing application and transport
-          protocols, such as the [[[Interledger]]].
+        <li>Compatible with existing monetization protocols, such as the
```

Comment:
Technically, Interledger is a "protocol for sending packets of money across different payment networks or ledgers". I think monetization is a use case of it, but not a category.